python/src/aiconfig/schema.py

```diff
@@ -79,6 +79,15 @@ class OutputDataWithToolCallsValue(BaseModel):
     OutputDataWithToolCallsValue,
 ]
 
+class AttachmentDataWithStringValue(BaseModel):
+    """
+    This represents the input data that is storied as a string, but we use
```

Comment:
nit, `input data that is storied` -> `attachment data that is stored` and should say 'the `mime_type` in Attachment'